plugin/auth_ldap/include/plugin_variables.h

```diff
@@ -39,6 +39,8 @@ static int log_status;
 static unsigned int max_pool_size;
 static char *server_host;
 static unsigned int server_port;
+static char *fallback_server_host;
```

Comment:
I agree, but the server_host and server_port variables are upstream, and we shouldn't change them, and I wanted to keep the fallbacks the same.

I can change it to "fallback_hosts", or just make so that the port is optional, and the user can specify multiple hosts in the host variable.

Reply:
This seems a bit hacky IMHO, like if I have a port set it looks just the one address otherwise we can have multiple. I would rather:
1) remove fallback_server_port entirely
2) rename `fallback_server_host` to `fallback_server_hosts`
3) populate the list of fallback servers based on `fallback_server_hosts` only